helm/ck-application/files/application.yaml

```
@@ -0,0 +1,56 @@
+logging:
```

yamllint

[error] wrong new line character: expected \n

(new-lines)